src/Hive.Core/sia380/sia380.py

```diff
@@ -1,7 +1,7 @@
 # coding=utf-8
 """
 Monthly heating, cooling and electricity demand calculation according to SIA 380.
-Variables names according to SIA 380.
+Equations from 'Interaktion Struktu, Klima und Wärmebedarf_HS20.xlsx', 19.10.2020
```

Comment:
what is this? I tried searching for it - it's not in the repo...

Reply:
added files into sia380.py folder
a166a0b208326f1e5b89678c69363254248248a3